lib/parallel_tests/cli.rb

```diff
@@ -192,7 +192,16 @@ def execute_shell_command_in_parallel(command, num_processes, options)
     def report_time_taken
       start = Time.now
       yield
-      puts "\nTook #{Time.now - start} seconds"
+      elapsed = Time.now - start
+      display_elapsed = display_duration(elapsed)
+      puts "\nTook #{elapsed} seconds" + (display_elapsed.include?(':') ? " (#{display_elapsed})" : "")
```

Comment:
I wanted to leave the existing output undisturbed (there's got to be someone, somewhere parsing it with a script...) and didn't want to annoy users (who have somehow managed to keep their test suites under a minute) with redundant display.